src/config.c

```diff
@@ -105,6 +106,13 @@ struct Keyword listener_stanza_grammar[] = {
             (int(*)(void *, char *))accept_listener_protocol,
             NULL,
             NULL},
+#ifdef SO_REUSEPORT
```

Comment:
Do we want to modify the configuration grammar on platforms without SO_REUSEPORT or reject a configuration with `reuseport` turned on when the platform doesn't support SO_REUSEPORT?